bin/start-uc-server

```diff
@@ -6,6 +6,14 @@ ROOT_DIR="$(dirname "$SCRIPT_DIR")"
 TARBALL_JAR_DIR="$ROOT_DIR/jars"
 SERVER_CLASS_FILE="$ROOT_DIR/server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java"
 
+# Use DEBUG env var to turn remote Java debugging (JDWP) on
+# DEBUG is the port that the JDWP agent listens to
+DEBUG_CMD=""
+if [[ -n "$DEBUG" ]]; then
```

Comment:
Can we name DEBUG to DEBUG_PORT to be more explicit?

Reply:
"Great minds think alike" here 😉  That's exactly what I tried the very first time but thought UC devs won't like typing so much and although not very explanatory it's not supposed to be so. It's for the devs after all, and the shorter (and even cryptic) the better.

After I ended up with `DEBUG` I liked it very much and would be against expanding it to include `_PORT` until it's really really needed (e.g., when we decide to add even more debug-related env vars).

Could this explanation be OK to convince you to accept `DEBUG` @yaoderek, please? 🙏

Reply:
Thanks for the explanation @jaceklaskowski . If we want to use DEBUG, can we make it a flag with optional value (5001 by default). This way, the cmd can be even short like.

`DEBUG ./bin/start-uc-server`